content/applications/websites/livechat.rst

```diff
@@ -1,85 +1,105 @@
+Odoo *Live Chat* allows users to communicate with website visitors in real-time. With *Live Chat*,
 leads can be qualified for their sales potential, support questions can be answered quickly, and
-issues can be directed to the appropriate team for further investigation (or follow up). *Live Chat*
+issues can be directed to the appropriate team for further investigation or follow up. *Live Chat*
 also provides the opportunity for instant feedback from customers.
 
 Enable Live Chat
 ================
 
-In order to enable *Live Chat*, the *Live Chat* application needs to be installed. This can be done
-in one of two ways.
+The *Live Chat* application can be installed multiple ways:
 
-- Go to :menuselection:`Apps --> Live Chat` and click :guilabel:`Install`.
-- In the :menuselection:`Website` application, go to :menuselection:`Configuration --> Settings`,
-  scroll to the :guilabel:`Email & Marketing` section, check the box next to :guilabel:`Livechat`,
-  and click :guilabel:`Save`.
+- Go to :menuselection:`Apps application --> Live Chat` and click :guilabel:`Install`.
+- Go to the :menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams` list view, select a
+  team, and on the team's settings page, click the checkbox next to :guilabel:`Live Chat`, under the
+  :guilabel:`Channels` section.
+- In the :menuselection:`Website` app, go to :menuselection:`Configuration --> Settings`, scroll to
+  the :guilabel:`Email & Marketing` section, check the box next to :guilabel:`Livechat`, and click
+  :guilabel:`Save`.
 
 .. image:: livechat/enable-setting.png
    :align: center
    :alt: View of the settings page and the live chat feature for Odoo Live Chat.
 
-After the :guilabel:`Live Chat` application is installed, a live chat :guilabel:`Channel` will be
-created by default and automatically selected in the drop-down.
+.. note::
+   After the *Live Chat* application is installed, a live chat *Channel* is created by default.
 
-Create a new live chat channel
-==============================
+Create live chat channels
+=========================
 
 To create a new live chat *Channel*, go to :menuselection:`Main Odoo Dashboard --> Live Chat app -->
-New`. This will open a blank channel detail form. Enter the name of the new channel in the
+New`. This opens a blank channel detail form. Enter the name of the new channel in the
 :guilabel:`Channel Name` field.
 
 .. image:: livechat/open-channel.png
    :align: center
    :alt: View of a live chat channel form for Odoo Live Chat.
 
-To configure the remaining tabs on the channel detail form (:guilabel:`Operators`,
-:guilabel:`Options`, :guilabel:`Channel Rules`, and :guilabel:`Widgets`), follow the steps below.
+To configure the remaining tabs on the channel detail form (:ref:`Operators <operators-tab>`,
+:ref:`Options <options-tab>`, :ref:`Channel Rules <channel-rules-tab>`, and :ref:`Widget
+<widget-tab>`), follow the steps below.
```

Comment:
Please include the app/module name in the custom anchor declarations and references
```suggestion
To configure the remaining tabs on the channel detail form (:ref:`Operators
<livechat/operators-tab>`, :ref:`Options <livechat/options-tab>`, :ref:`Channel Rules
<livechat/channel-rules-tab>`, and :ref:`Widget <livechat/widget-tab>`), follow the steps below.
```